diff --git a/dist/db/get-migration.mjs b/dist/db/get-migration.mjs
index 0000000..0000001 100644
--- a/dist/db/get-migration.mjs
+++ b/dist/db/get-migration.mjs
@@ -131,7 +131,7 @@
 				FROM sqlite_master AS tables
 				INNER JOIN pragma_index_list(tables.name) AS index_list
 				INNER JOIN pragma_index_info(index_list.name) AS index_info
-				WHERE tables.type = 'table'
+				WHERE tables.type = 'table' AND substr(tables.name, 1, 1) <> '_'
 			`.execute(db)).rows;
 			break;
 		case "postgres":
